.circleci/config.yml

```diff
@@ -472,13 +472,24 @@ jobs:
                     sed -i.bak "s|${OPERATOR_VERSION}|${LATEST_TAG}|g" ./test/fixtures/operator/catalog-source.yaml
                     kubectl apply -f ./test/fixtures/operator/catalog-source.yaml
 
-                    sleep 120
+                    ATTEMPTS=60
+                    SLEEP_SECONDS_BETWEEN_ATTEMPTS=5
+                    # total = 5 minutes wait time
 
-                    VERSION=$(kubectl get pods -n snyk-monitor --no-headers | \
-                      grep "snyk-monitor" | \
-                      awk 'END { if (NR==0) exit 1; else print $1 }' | \
-                      xargs -I{} kubectl get pod {} -n snyk-monitor -o jsonpath={..containers[*].image} | \
-                      awk '{print $1}' | grep -oE "[0-9]{1}\.[0-9]{1,2}\.[0-9]{1,3}$")
+                    for (( attempt=1; attempt<ATTEMPTS; attempt++))
+                    do
+                      VERSION=$(kubectl get pods -n snyk-monitor --no-headers | \
```

Comment:
I know that might be trivial, but can you ask more comments about what each command does, so it would be easy to fix if smth change in the future, please?